weave-js/src/components/RadixTooltip/index.tsx

```diff
@@ -57,7 +57,7 @@ export const Portal = RadixTooltip.Portal;
  */
 export const Content = React.forwardRef(
   ({className, children, ...props}: RadixTooltip.TooltipContentProps, ref) => (
-    <Tailwind>
+    <Tailwind style={{display: 'contents'}}>
```

Comment:
side note: i'm starting to think we should just add `display: contents` on the Tailwind wrapper by default because we actually do this quite a lot throughout the application